tests/test_manylinux.py

```diff
@@ -225,6 +225,66 @@ def test_build_wheel_with_binary_executable(docker_container):
         python_id, ['python', '-c', 'from testpackage import runit; print(runit(1.5))']).strip()
     assert output.strip() == '2.25'
 
+@pytest.mark.parametrize('with_dependency', ['0', '1'])
+def test_build_wheel_with_highest_policy_and_image_dependencies(with_dependency, docker_container):
+    # Always try to repair the wheel targeting manylinux1 policy
+    #
+    # with_dependency == 0
+    #   The python module has no dependencies that should be grafted-in and
+    #   uses versioned symbols not available on policies pre-dating the policy matching the image being tested.
+    # with_dependency == 1
+    #   The python module itself does not use versioned symbols but has a dependency that should be grafted-in that
+    #   uses versioned symbols not available on policies pre-dating the policy matching the image being tested.
+    # Since manylinux1 has no pre-dating policy, it's the only image/policy that should be able to repair those wheels.
```

Comment:
This sentence doesn't quite match my understanding of how the test works.

We can repair the wheel built on any image, but only to the target platform corresponding to the image. The test checks that a wheel built on manylinux2010 can be repaired to target manylinux1, and that a wheel built on manylinux1 can be.